explorer/explorerroutes.go

```diff
@@ -477,6 +477,12 @@ func (exp *explorerUI) timeBasedBlocksListing(val string, w http.ResponseWriter,
 		return
 	}
 
+	// If the view is "years" and the top row is this year, modify the formatted
+	// time string to indicate its a partial result.
+	if val == "Years" && len(data) > 0 && data[0].EndTime.T.Year() == time.Now().Year() {
+		data[0].FormattedStartTime = fmt.Sprintf("%s so far", time.Now().Format("2006"))
```

Comment:
Doing it this way prevents the need for a wide "end date" column on /years.

Reply:
Makes a lot of sense.  I'm just wondering if YTD or "year to date" might be clearer than say "2019 so far".  I'm OK with this though.

Reply:
Year to date is much better. Should it be hyphenated, "year-to-date" or not, "year to date"?

Reply:
Good with YTD as you set it.  Let's see if we get comments when live.